com.modrinth.ModrinthApp.yml

```diff
@@ -87,7 +87,7 @@ modules:
         commit: e9e2e37c329dbe98bfeb27a1828ef9a71fa84f88
         x-checker-data:
           type: git
-          tag-pattern: ^v([\\d.]+)$
+          tag-pattern: "^v([\\d.]+)$"
```

Comment:
```suggestion
          tag-pattern: ^v([\d.]+)$
```

Reply:
You don't need to escape with `\` in yaml. Quoting it doesn't fix it, it still stalls.